projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/RepositoryUtil.java

```diff
@@ -151,4 +152,14 @@ public static DataJobExecution createDataJobExecution(
 
     return jobExecutionRepository.save(expectedJobExecution);
   }
+
+  /**
+   * at the database level we only store date-times accurate to the microsecond. Like wise in older
+   * versions of java .now() returned timestamps accurate to micro-seconds. In newer versions of
+   * java .now() gives nano-second precision and it causes tests written before we adopted that java
+   * version to fail.
+   */
+  public static OffsetDateTime getTimeAccurateToMicroSecond() {
```

Comment:
I find the documentation great. My only concern is with duplicated methods
with JobExecutionUtil